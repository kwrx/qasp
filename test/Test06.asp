% Test with iterations

%@exists
b :- a.
e | f.
%@exists
h :- b.
a.
%@constraints
{b;h}.
:- b, h.

