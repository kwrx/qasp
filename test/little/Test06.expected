QASP 1.0

{e}
{f}
